### ISSUE TITLE
User dropdown menu was disabled in Reports result page

### ISSUE THREAD

**Issue by atom-tr** [netbox-community/netbox#7962]
### NetBox version

v3.0.11

### Python version

3.9

### Steps to Reproduce

1. go to Reports
2. Open a Report result
3. Can't go to Setting from this page
4. Console.log show: _Table does not contain enable/disable toggle buttons_

### Expected Behavior

Can go to setting anywhere if it possible.

### Observed Behavior

User dropdown menu was disabled.

**Comment by jeremystretch** [comment]
Thank you for opening a bug report. Unfortunately, the information you have provided is not sufficient for someone else to attempt to reproduce the reported behavior. Remember, each bug report must include detailed steps that someone else can follow on a clean, empty NetBox installation to reproduce the exact problem you're experiencing. These instructions should include the creation of any involved objects, any configuration changes, and complete accounting of the actions being taken. Also be sure that your report does not reference data on the public NetBox demo, as that is subject to change at any time by an outside party and cannot be relied upon for bug reports.

**Comment by kkthxbye-code** [comment]
I actually have run into this issue, it's the same with custom scripts. When you are on any result page, the top right dropdown menu does not work.

Steps:

1.  Create testreport.py in the reports dir.
```
from extras.reports import Report

class TestReport(Report):
    description = "test"

    def test_test(self):
        self.log_failure(None, "test")
```

2. Go to Other -> Reports and execute the report
3. You will be redirected to the results page, and here the top right dropdown menu will not work

I debugged it a little. It breaks because of the jobs.js file. Specifically these three lines, if commented out, makes the dropdown menu functional again:

https://github.com/netbox-community/netbox/blob/83010e278c3046a48368f2c5d7ca6bd3ee38e8ff/netbox/project-static/src/jobs.ts#L68-L71

I can't find the reason though, I have no idea how it breaks it. If you make the code unreachable, by changing the conditional to `if (false)` here:

https://github.com/netbox-community/netbox/blob/83010e278c3046a48368f2c5d7ca6bd3ee38e8ff/netbox/project-static/src/jobs.ts#L67-L72

It still breaks the dropdown. So simply having a call to createToast in the code breaks the dropdown. @thatmattlove probably needs to take a look at it, because to me it makes no sense.

**Comment by jeremystretch** [comment]
I'm currently working on #8114 to replace the job handling Javascript entirely with HTMX. This should be resolved once that work has been merged. Going to mark this as blocked for now.

**Comment by jeremystretch** [comment]
:+1: This has been fixed under #8114.
